ktor-features/ktor-auth-jwt/src/io/ktor/auth/jwt/JWTAuth.kt

```diff
@@ -107,8 +116,8 @@ private suspend fun verifyAndValidate(call: ApplicationCall, jwtVerifier: JWTVer
     return validate(call, credentials)
 }
 
-private fun HttpAuthHeader?.getBlob() = when {
-    this is HttpAuthHeader.Single && authScheme == "Bearer" -> blob
+private fun HttpAuthHeader?.getBlob(supportedAuthSchemes: Set<String>) = when {
+    this is HttpAuthHeader.Single && authScheme in supportedAuthSchemes -> blob
```

Comment:
should it be case sensitive?

Reply:
Yes. It seems so: https://tools.ietf.org/html/rfc7235#section-2.1